src/libstore/s3-binary-cache-store.cc

```diff
@@ -159,7 +159,11 @@ ref<Aws::Client::ClientConfiguration> S3Helper::makeConfig(
 S3Helper::FileTransferResult S3Helper::getObject(
     const std::string & bucketName, const std::string & key)
 {
-    debug("fetching 's3://%s/%s'...", bucketName, key);
+    std::string uri = "s3://" + bucketName + "/" + key;
+    Activity act(*logger, lvlTalkative, actFileTransfer,
+        fmt("downloading '%s'", uri),
+        Logger::Fields{uri}, getCurActivity());
+    debug("fetching '%s'...", uri);
```

Comment:
```suggestion
```
Superfluous since we already have an activity that basically says the same thing.